### DISCUSSION TITLE
Active maintainer

### DISCUSSION THREAD

**Discussion by BartInTheField** [serverless/serverless-azure-functions#643] [General]
Hi @medikoo, 

I don't know if this is the right place to reach out to you. I have seen the announcement in the README for quite some time "Currently looking for active maintainers". Currently we are using this plugin for a customer and we have done some adjustments to support node 16 and node 18 using `npx patch-package`. 

I would like to merge these changes back to the official plugin, but there is currently no active maintainer. Because we would like to stay using this plugin in for our customer I would like to be maintainer of the plugin so people can stay working on azure using serverless.

**Comment by medikoo** [comment]
@BartInTheField please propose a PR, I can merge it, and if you'd share further interest in maintaining the project we can certainly provide you with collaboration rights

**Reply by BartInTheField** [reply]
I see someone beat me to it, I have seen multiple pull requests like this one. But none seem to be merged

https://github.com/serverless/serverless-azure-functions/pull/642

**Comment by gligorkot** [comment]
@medikoo, I'd be happy to take on a more active maintainer role on this repo if you're happy to share the load. We're using this plugin actively on a production environment so keeping this one up to date would make it a lot better for us, and if we can do it ourselves even better!

**Reply by medikoo** [reply]
@gligorkot, great to hear that! I've just merged your PR. It looks like a significant step forward for this project. Therefore I provided you with write rights to the repository - if you'd need to tweak any administration settings, ping me.

I've also configured `USER_GITHUB_TOKEN` and `NPM_TOKEN` secrets, so via CI, we should be able to publish new version tags and push new releases to npm. Let me know if anything will require further looking into it (setting up CI for the first time often turns bumpy)

**Reply by gligorkot** [reply]
Thanks @medikoo that sounds great!

**Reply by gligorkot** [reply]
@medikoo, it looks like the serverless-ci user doesn't have permissions to push a tag to the repo. Can you please help with this one, then I can manage the merging of PRs and release any new versions. Thanks!

**Reply by gligorkot** [reply]
Also @medikoo, I believe there is a protection on the master branch so I wouldn't be able to merge without a review. Would need at least one other active maintainer to approve the PRs or we remove the necessity to have a review before merging it?

**Reply by medikoo** [reply]
Indeed, I forgot to provide it with the necessary rights. I've just fixed it, and pushed the v2.2.0 tag manually

**Reply by medikoo** [reply]
> Also @medikoo, I believe there is a protection on the master branch so I wouldn't be able to merge without a review.

Good point. I've just relaxed this requirement
